CHANGELOG.md

```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
```

Comment:
The brackets in keepachangelog are just links.

Reply:
I referred to https://keepachangelog.com/en/1.0.0/. The example uses the brackets for Unreleased.
What do you think?

Reply:
They are reference links (markdown), scroll to the end of the "file".

Reply:
Oops, That's right. I'll change Unreleased and 0.2.0. In addition, I'll remove `Deprecated`.